### DISCUSSION TITLE
group by and distinct

### DISCUSSION THREAD

**Discussion by fafo** [collerek/ormar#192] [Q&A]
Hi, 
from the documentation I don't get how to get distinct values in a column or count values grouped by a value. Please can you help with an example?

**Comment by collerek** [comment]
Hi, That's quite a low level sql query and ormar do not support this as of now.
But you can still issue a query generated from Models with help of sqlalchemy through databases.

If you provide Models you have and what you want to achieve exactly (like group by what, distinct from which column etc.) I can help you to construct such query.

**Reply by fafo** [reply]
No problems, thank you. I manually issued the sql query, since it was a bit complex. I had a set of updates for several objects and I needed the last update of each object: a manual select was the way, then I built the models with the results (what I love about ormar is the unique db/pydantic model).   
I was just wandering if there was an idiomatic way using ormar and I was missing it.

**Comment by sorasful** [comment]
Hi @collerek  ! 

I'm curious on how you would proceed, given a model `Image` (for example) and want to get 30 results but with distinct attribute `link` .

Thank you !
